forc-pkg/src/pkg.rs

```diff
@@ -92,14 +93,41 @@ pub struct BuiltPackage {
     pub json_abi_program: ProgramABI,
     pub storage_slots: Vec<StorageSlot>,
     pub bytecode: Vec<u8>,
-    pub entries: Vec<FinalizedEntry>,
+    pub pkg_entries: Vec<PkgEntry>,
```

Comment:
I think `entries` is fine as a field name, as the association with pkg is inferred from both the type and likely also the variable names when it's used, e.g.
```rust
built_pkg.entries
```
rather than
```rust
built_pkg.pkg_entries
```